### DISCUSSION TITLE
WithSecure Problem

### DISCUSSION THREAD

**Discussion by AnuBens** [Estia-1a/projetGenieInfo_public#81] [Project Installation and Configuration] [answered]
**Bug description**

L'antivirus met en quarantaine le fichier freud.exe ou d'autres fichiers produits par vscode. 

**Step to solve**
Il suffit d'aller dans l'application WithSecure Elements Agent (en bas à coté de l'horloge)
- puis dans " Protection anti-programmes malveillants " 
- et "Afficher la quarantaine " rajouter le chemin d'accès de votre fichier où est localisé  freud.exe afin que l'antivirus n'ait pas à y faire de vérification.

Vous n'avez plus que à run votre programme et le problème est réglé!

**Comment by dhmmasson** [comment, marked as answer]
Il suffit d'aller dans l'application WithSecure Elements Agent (en bas à coté de l'horloge)

puis dans " Protection anti-programmes malveillants "
et "Afficher la quarantaine " rajouter le chemin d'accès de votre fichier où est localisé freud.exe afin que l'antivirus n'ait pas à y faire de vérification.
Vous n'avez plus que à run votre programme et le problème est réglé!
